/**:
  ros__parameters:
    num_vehicles: 4